biz.aQute.bndlib/src/aQute/bnd/service/result/FunctionWithException.java

```diff
@@ -0,0 +1,74 @@
+package aQute.bnd.service.result;
```

Comment:
We already have https://github.com/bndtools/bnd/blob/master/aQute.libg/src/aQute/lib/exceptions/FunctionWithException.java

Reply:
The libg is private, it cannot be used in public API